autokeras/tasks/image.py

```diff
@@ -221,3 +221,103 @@ def fit(self,
                     validation_split=validation_split,
                     validation_data=validation_data,
                     **kwargs)
+
+
+class ImageSegmenter(SupervisedImagePipeline):
+    """AutoKeras image segmentation class.
+
+    # Arguments
+        num_classes: Int. Defaults to None. If None, it will infer from the data.
+        loss: A loss function. Defaults to use 'tf.losses.softmax_cross_entropy'.
+        metrics: A list of metrics used to measure the accuracy of the model,
+        default to 'mIoU'
```

Comment:
1. Add an indent here.
2. a period in the end.
3. Let's use accuracy as default value.